lib/DependencyScan/ModuleDependencyCacheSerialization.cpp

```diff
@@ -1732,20 +1736,17 @@ void ModuleDependenciesCacheSerializer::collectStringsAndArrays(
   for (auto kind = ModuleDependencyKind::FirstKind;
        kind != ModuleDependencyKind::LastKind; ++kind) {
     auto modMap = cache.getDependenciesMap(kind);
-    for (const auto &entry : modMap) {
-      ModuleDependencyID moduleID = {entry.getKey().str(), kind};
+    std::vector<StringRef> sortedKeys;
```

Comment:
Using initializer `{ modMap.keys().begin(), modMap.keys().end() }` is cleaner here.